README.org

```diff
@@ -18,3 +18,6 @@ Here's a table describing the bindings
 | =ediff-suspend=             | =z=              | =C-z=          |
 | scroll left                 | =>=              | =zh=           |
 | scroll right                | =<=              | =zl=           |
+| copy A region to B's region | a                | l              |
+| copy B region to A's region | b                | h              |
```

Comment:
i can also image `H` and `L` for copying a single line instead of the highlighted block,
that could work if the `j` and `k` would actually change the selection to a single line and move the that selection to the next/previous line, instead of doing the jobs of C-e and C-y
but I am probably dreaming already :]